src/ios/OpenTokPlugin.m

```diff
@@ -432,7 +445,9 @@ - (void)unsubscribe:(CDVInvokedUrlCommand*)command{
     NSString* sid = [command.arguments objectAtIndex:0];
     OTSubscriber * subscriber = [subscriberDictionary objectForKey:sid];
     @try {
-        [_session unsubscribe:subscriber error:nil];
+        NSString* sessionId = [command.arguments objectAtIndex:1];
```

Comment:
move to the beginning of the function next to all the others `[command.arguments objectAtIndex`